lib/presentation/crowd_action/pages/in_spot_light.dart

```diff
@@ -1,7 +1,7 @@
 import 'package:collaction_app/application/crowdaction/spotlight/spot_light_bloc.dart';
 import 'package:collaction_app/infrastructure/core/injection.dart';
 import 'package:collaction_app/presentation/crowd_action/pages/widgets/share_collaction_card.dart';
-import 'package:collaction_app/presentation/home/widgets/current_upcoming_ca_layout.dart';
+import 'package:collaction_app/presentation/home/widgets/current_upcoming_layout.dart';
```

Comment:
Use relative imports for all **package:collaction_app** imports

Reply:
@wizlif should I replace with relative imports for all 4 lines?